packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.2.3
+
+* Added `setScrollListener` method to the `AndroidWebViewController`.
```

Comment:
Ios can also support listening to scroll events, so we should add the method to the platform interface and implement for both platforms. This will probably change to something like "Implements support for `PlatformWebViewController.setOnScrollChange`".